rpc/client/client_test.go

```diff
@@ -214,6 +215,77 @@ func TestGetBlock(t *testing.T) {
 	require.NoError(err)
 }
 
+func TestGetCommit(t *testing.T) {
+	require := require.New(t)
+	mockApp, rpc := getRPC(t)
+	mockApp.On("BeginBlock", mock.Anything).Return(abci.ResponseBeginBlock{})
+	mockApp.On("Commit", mock.Anything).Return(abci.ResponseCommit{})
+
+	block := getRandomBlock(1, 5)
+
+	err := rpc.node.Start()
+	require.NoError(err)
+
+	err = rpc.node.Store.SaveBlock(block, &types.Commit{})
```

Comment:
Please add some data into commit and later assert that returned value is the same as expected.